index.js

```diff
@@ -47,16 +47,26 @@ function incrementVersionOfPackage(registryVersions, options) {
   const nextVersion = versionCalculations.calculateNextVersionPackage(localPackageVersion, registryVersions, options);
 
   if (nextVersion !== localPackageVersion) {
-    execSync(`npm version --no-git-tag-version ${nextVersion}`, {cwd: options.cwd});
+    writePackageVersion(nextVersion, options.cwd);
   }
   return nextVersion;
 }
 
+function writePackageVersion(version, cwd) {
+  execSync(`npm version --no-git-tag-version ${version}`, {cwd});
+}
+
 function prepareForRelease(options) {
   options = options || {};
   options.cwd = options.cwd || process.cwd();
 
   const pkg = packageHandler.readPackageJson(path.join(options.cwd, 'package.json'));
+  if (process.env.DANGEROUSLY_FORCE_PKG_VERSION) {
```

Comment:
Nice! shouldn't we do it after the `private` check?

Reply:
It doesn't matter IMO, since if the package is private there won't be a release anyway...
I'll move that.